workshops/how_we_enable/03-Strategy.md

```diff
@@ -1,12 +1,51 @@
-# High Level Strategy Accelerates Success
+# Enterprise Architecture Standards Deep Dive
 
-## Why Strategy Matters
+## APIs
 
-"Formalizing your organization’s approach to open source management and strategy creates guidelines that boost efficiency and minimize risks. Whether or not you have set a business strategy around your open source efforts, you likely already know that this is important." - Linuxfoundation.org
+Application Programming Interfaces are the defacto standard for how applications talk to each other. Making these be secure, reusable and cataloging them so they are accessible to your customers is a standard practice for digital transformation.
 
-At the highest level having company standards around processes is one of the most important things to address in order to help accelerate the adoption of open source software and tools.
+_Note:_ "Customers" can be internal or external customers.
 
-### Source Code Management Strategy
+## Automation
+
+"Covid-19 has made automation a boardroom imperative.” - Forrester Research 2020
+Automating all services and making them consumable via APIs is the standard way of doing this in today's IT. Most people start with a command line tool, and build it into a pipeline or pass around cookbooks or playbooks, which is a great way to start, however someone needs to know where these are. A common practice is using a centralized tool like Ansible Tower to make all of these playbooks self service by exposing them as APIs.
```

Comment:
Maybe add a link to Ansible tower?

Reply:
fixed in 5054ffc9149b2fe9141cb0fc6307ec41cd1c41c6